lib/job.js

```diff
@@ -193,13 +193,25 @@ class Job extends BaseModel {
 
     /**
      * Return latest build that belong to this job
-     * @param  {Object}   [config]                  Configuration object
-     * @param  {String}   [config.status]           Return latest build with this status
-     * @return {Promise}                            Lastest build
+     * @param  {Object}   [config]              Configuration object
+     * @param  {String}   [config.status]       Return build with this status
+     * @param  {String}   [config.position]     Return build with this position relative to latest build
+     * @return {Promise}                        Lastest build
      */
     getLatestBuild(config = {}) {
-        return this.getBuilds({ status: config.status })
-            .then(latestBuilds => latestBuilds[0] || {});
+        const { position, status } = config;
+        // passing string instead of number to retain old behavior
+        const parsedPoistion = Number(position);
+        let relativePosition = -1;
+
+        if (parsedPoistion && parsedPoistion > 0) {
+            relativePosition = parsedPoistion;
+        } else if (position === 'latestBuild') { // to retain old behavior
```

Comment:
> `position === 'latestBuild'`

If this is user input that passed from url, could we change `latestBuild` to `latest`? 

Also, I don't have much comment on `position` used here, or its synonyms: order, revision.
Not sure what term people usually use.

@jithine @tkyi Anyone suggestions on the naming?

Reply:
As comment suggest, this is to retain old behavior. So this change does not break any current build that happen to rely on previous behavior. Changing it to latest WILL BREAK IT